src/Router/Artsy2.tsx

```diff
@@ -0,0 +1,48 @@
+import { ContextProps as _ContextProps } from "Components/Artsy"
+import React, { SFC } from "react"
+import { getUser } from "Utils/getUser"
+import { createEnvironment } from "../Relay/createEnvironment"
+
+// FIXME: Move from Artsy to Artsy2
+export interface ContextProps extends _ContextProps {
+  // FIXME: Type properly
+  mediator?: any
```

Comment:
Please do type this appropriately.

Reply:
And is it really optional? Seems like it’s something all apps depend on, no?

Reply:
In the future we should be able to phase it out, but for the time being it does seem necessary since thats how we trigger logins and so on. Will update.